docs/sources/tempo/configuration/parquet.md

```diff
@@ -32,14 +32,14 @@ You can still use the previous format `vParquet2`.
 To enable it, set the block version option to `vParquet2` in the Storage section of the configuration file.
```

Comment:
this is weird advice. we should likely not be advising anyone to set their version to vParquet2 at this point.

Reply:
Well, the full context was. Default = vParquet3. But "You can still use the previous format `vParquet2`."